Formula/otto.rb

```diff
@@ -79,9 +79,12 @@ def install
 
     ENV["GOPATH"] = gopath
     ENV.prepend_create_path "PATH", gopath/"bin"
-
     Language::Go.stage_deps resources, gopath/"src"
 
+    # Fix for Go 1.7+ syntax strictness. Upstream is dead.
```

Comment:
I'm not sure. Typically for simple syntax errors like this we've fixed it and reported it upstream even where upstream hasn't committed in upwards of 6 months, but upstream has hard deprecated the repo, there's almost nothing in it now, you have to go back to the old tag to find the contents.

I'm open to the simple fix or death either way, but there don't seem to be any issues beyond the lack of `=` in the prior tag.